### PR TITLE
Use new favicon

### PR DESCRIPTION
**Pull request by kul3r4**
@rowan-m @jmadler @sebastianbenz AMP icon has been updated, this would be the new favicon for ABE

### CODE REVIEW

**Comment by rowan-m** [comment]
Will also need updated preview for https://github.com/ampproject/amp-by-example/blob/master/src/img/abe_preview.png

**Comment by kul3r4** [comment]
Thanks, all done!

**Comment by sebastianbenz** [comment]
The lightning bolt should be white instead of transparent.

**Comment by kul3r4** [comment]
@sebastianbenz lightning bolt is now white. Thanks!

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

File: src/img/ABE-logo.svg [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (sebastianbenz) [inline src/img/ABE-logo.svg]: rename file to logo.svg

File: templates/css/styles.css [hunk 2]
```diff
@@ -509,12 +509,11 @@ footer ul li a {
     display: none;
   }
   header #logo {
-    width: auto;
+    width: 200px;
     margin: 0;
     padding: 0;
     font-weight: 400;
```
Comment (sebastianbenz) [inline templates/css/styles.css]: remove font related rules as they're no longer needed

File: templates/css/styles.css [hunk 3]
```diff
@@ -245,14 +245,14 @@ header #logo {
   text-align: left;
   font-weight: 300;
```
Comment (sebastianbenz) [inline templates/css/styles.css]: remove font related rules as they're no longer needed

**Comment by sebastianbenz** [comment]
Applied the remaining changes while merging.

**Pull request merged**